stacker_blueprints/ecs.py

```diff
@@ -398,6 +418,46 @@ def create_role(self):
         self.add_output("RoleArn", self.role.GetAtt("Arn"))
         self.add_output("RoleId", self.role.GetAtt("RoleId"))
 
+    def create_task_execution_role(self):
```

Comment:
So this actually won't work in the EC2 version of ECS.  TaskExecutionRoles are meant to be used with Fargate from my understanding, and with EC2 the effective task execution role is the one that is applied to your Container Instance's (EC2) InstanceProfile.

So basically we just need to include the policy statements that you would normally include here in the Instance role/policy (which isn't something we can do here).

Reply:
See:

https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_execution_IAM_role.html
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/instance_IAM_role.html